app/navigation/Navigator.scala

```diff
@@ -26,8 +26,9 @@ import models._
 @Singleton
 class Navigator @Inject() () {
 
-  private val normalRoutes: Page => UserAnswers => Call = { case _ =>
-    _ => routes.ProfileSetupController.onPageLoad
+  private val normalRoutes: Page => UserAnswers => Call = {
```

Comment:
We can define this within the controller itself, and from my experience that makes it a lot easier to follow and update from a development perspective. Could you please remove this Navigator class?

Reply:
This is removed along with FakeNavigator/Spec